docker-compose.yaml

```diff
@@ -98,12 +98,12 @@ services:
       options:
         max-size: "20m"
     build:
-      context: ./
+      context: ./scripts/generate-accounts
       dockerfile: Dockerfile.generateAccounts
     volumes:
-      - ./accounts:/app/accounts
-      - ./scripts/keys:/keys
-    command: npm run generate-accounts
+      - ./scripts/generate-accounts/accounts:/app/accounts
+      - ./scripts/generate-accounts/keys:/app/keys
+    command: npx ts-node index.ts
```

Comment:
Command from `package.json` should be executed here.

Reply:
Updated.